src/components/atoms/Switch/Switch.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react'
+import styled from 'styled-components'
+
+// I N T E R F A C E
+export interface SwitchProps {
+  border?: boolean
+  checked?: boolean
+  className?: string
+  label?: string
+  onChange?: () => void
+  style?: React.CSSProperties
+  type?: SwitchType
+}
+
+// T Y P E S
+export type SwitchType = 'slim' | 'bold'
+
+export const Switch: React.FC<SwitchProps> = ({
+  border = false,
+  checked,
+  className,
+  onChange,
+  style,
+  type = 'slim'
+}) => {
+  return (
+    <StyledSwitchWrapper
+      className={`switch ${className}`}
+      style={style}
+    >
+      <input
+        className='switch-checkbox'
+        id='switch-new'
```

Comment:
wouldn't this cause invalid html if you use multiple switches on one page?

Reply:
It did, thanks for the hint